doc/news/_preparation_next_release.md

```diff
@@ -67,11 +67,43 @@ You can also read the [new low-level docs](../dev/kdb-operations.md) to find out
 The structure of `system:/elektra/mountpoints` changed as well.
 Take a look at the [new docs](../dev/mountpoints.md), if you need to know details.
 
+#### Updating config
+
+> **Warning**: BACK UP YOUR CONFIG FILES BEFORE UPDATING!
+
+To update your existing `system:/elektra/mountpoints` data you can use the [migration script](/scripts/migrate-mountpoints.py).
+
+> **Note**: To run the script you must have Elektra and the Python binding installed.
+> The script uses the Python binding to manipulate `Key`s and `KeySet`s, but it does not the `kdb` CLI tool, or the `KDB` API.
+> It is safe to run this script before, or after you update your Elektra installation.
+
+By default, the script loads the file `/etc/kdb/elektra.ecf`.
+If you changed where `system:/elektra/mountpoints` is stored, you can provide an alternative path:
+
+```
+./migrate-mountpoint.py /path/to/your/mountpoints/config.file
+```
+
+> **Note**: Because the script does not use the `KDB` API it only works, if the mountpoints config file uses the default `dump` format.
```

Comment:
You could use `kdb convert` before?

Reply:
Yes, but it'll probably only work, if you run `kdb convert` before updating Elektra. After updating `kdb` probably crashes because it can't load it's own config.